diff --git index.js index.js
index 4853e2a..08e26cb 100644
--- index.js
+++ index.js
@@ -56,17 +56,18 @@ module.exports = function (target, dest, opts = {}) {
 
 // Installs the package specified by target and returns the installation path
 function installPackage (target, dest, opts) {
+    const { name, rawSpec } = npa(target, dest);
+
     return isNpmInstalled()
         // Ensure that `npm` installs to `dest` and not any of its ancestors
         .then(_ => fs.ensureDir(path.join(dest, 'node_modules')))
 
         // Run `npm` to install requested package
-        .then(_ => npmArgs(target, opts))
+        .then(_ => npmArgs(name, opts))
         .then(args => {
-            events.emit('verbose', `fetch: Installing ${target} to ${dest}`);
-            return superspawn.spawn('npm', args, { cwd: dest });
+            events.emit('verbose', `fetch: Installing ${name} to ${dest}`);
+            return superspawn.spawn('npm', args, { cwd: dest, stdio: 'inherit' });
         })
-
         // Resolve path to installed package
         .then(getTargetPackageSpecFromNpmInstallOutput)
         .then(spec => pathToInstalledPackage(spec, dest));
@@ -75,18 +76,18 @@ function installPackage (target, dest, opts) {
 function npmArgs (target, userOptions) {
     const opts = Object.assign({ production: true }, userOptions);
 
-    const args = ['install', target];
-
-    if (opts.production) {
-        args.push('--production');
-    }
-    if (opts.save_exact) {
-        args.push('--save-exact');
-    } else if (opts.save) {
-        args.push('--save');
-    } else {
-        args.push('--no-save');
-    }
+    const args = ['-ddd', 'install', target];
+
+    //if (opts.production) {
+    //    args.push('--production');
+    //}
+    //if (opts.save_exact) {
+    //    args.push('--save-exact');
+    //} else if (opts.save) {
+    //    args.push('--save');
+    //} else {
+    //    args.push('--no-save');
+    //}
     return args;
 }
 
