diff --git index.js index.js
index 4853e2a..a5be094 100644
--- index.js
+++ index.js
@@ -64,9 +64,8 @@ function installPackage (target, dest, opts) {
         .then(_ => npmArgs(target, opts))
         .then(args => {
             events.emit('verbose', `fetch: Installing ${target} to ${dest}`);
-            return superspawn.spawn('npm', args, { cwd: dest });
+            return superspawn.spawn('npm', args, { cwd: dest, stdio: 'inherit' });
         })
-
         // Resolve path to installed package
         .then(getTargetPackageSpecFromNpmInstallOutput)
         .then(spec => pathToInstalledPackage(spec, dest));
@@ -75,7 +74,7 @@ function installPackage (target, dest, opts) {
 function npmArgs (target, userOptions) {
     const opts = Object.assign({ production: true }, userOptions);
 
-    const args = ['install', target];
+    const args = ['--verbose', 'install', target];
 
     if (opts.production) {
         args.push('--production');
